internal/core/core_fence.go

```diff
@@ -0,0 +1,79 @@
+package core
+
+/*
+#cgo LDFLAGS: -lopenvds
+#cgo CXXFLAGS: -std=c++17
```

Comment:
I think we cam get rid of these in slice, fence and attributes files and leave them in core.go one only? At least it still compiles if I remove these two lines.